app/models/miq_expression.rb

```diff
@@ -430,11 +430,11 @@ def self._to_human(exp, options = {})
 
   def to_ruby(tz = nil)
     tz ||= "UTC"
-    @ruby ||= self.class._to_ruby(@exp.deep_clone, @context_type, tz)
+    @ruby ||= _to_ruby(@exp.deep_clone, @context_type, tz)
     @ruby.dup
   end
 
-  def self._to_ruby(exp, context_type, tz)
+  def _to_ruby(exp, context_type, tz)
```

Comment:
@yrudman just as a kindness, please use backticks or similar to avoid referencing me when writing @exp (it happens quite often!)